lib/_http_outgoing.js

```diff
@@ -423,8 +441,11 @@ OutgoingMessage.prototype.write = function(chunk, encoding, callback) {
   }
 
   if (!this._hasBody) {
-    debug('This type of response MUST NOT have a body. ' +
-          'Ignoring write() calls.');
+    err = new Error('this type of response must not have a body');
```

Comment:
"this type of response" is rather vague, could we change it to anything that is more concise?